src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.html

```diff
@@ -22,6 +22,23 @@
             >
             </jhi-tutor-participation-graph>
         </div>
+        <div class="col-lg-3 col-md-6 col-sm-6 align-self-center" *ngIf="isExamMode">
```

Comment:
This is duplicated code compared with assessment-dashboard.component.html 
Please create a small Angular component for the second correction toggle button and simply insert the button here and in the other location to avoid the code duplication

Reply:
Done 👍